### PR TITLE
updated readed for typo in configure

### PR DESCRIPTION
**Pull request by PatrickAlphaC**
This addresses https://github.com/smartcontractkit/chainlink/issues/3857

### CODE REVIEW

**Review by alexroan — APPROVED** [review]

**Review by se3000 — APPROVED** [review]

**Comment by PatrickAlphaC** [comment]
Closing in favor of https://github.com/smartcontractkit/chainlink/pull/3885

File: README.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Check out the [docs'](https://docs.chain.link/) pages on [Adapters](https://docs
 
 ## Configure
 
-You can configure your node's behavior by setting environment variables which can be, along with default values that get used if no corresponding environment variable is found. The latest information on configuration variables are available in [the docs](https://docs.chain.link/docs/configuration-variables).
+You can configure your node's behavior by setting environment variables. All the environment variables can be foudn in the `ConfigSchema` struct of `schema.go`, and you can read the [official documentation](https://docs.chain.link/docs/configuration-variables) to learn the most up to date information on each of them. For every variable, default values get used if no corresponding environment variable is found.
```
Comment (sexylexy69) [inline README.md:96]: Help please

Comment (sexylexy69) [inline README.md:96]: _watch & learn_

Comment (PatrickAlphaC) [reply README.md:96]: Want to jump into the discord to ask your questions? https://discord.gg/2YHSAey

**Pull request closed without merging**